web/src/app/modules/views/main/editors/modules/contents-container/components/test-specification-container.component.ts

```diff
@@ -28,8 +29,14 @@ export class TestSpecificationContainer extends ContentContainerBase<TestSpecifi
         navigator: NavigatorService,
         translate: TranslateService,
         modal: ConfirmationModal,
+        contentService: ContentsContainerService,
         clipboardService: ClipboardService) {
         super(dataService, navigator, translate, modal, clipboardService);
+
+        contentService.deleted$.subscribe(
+            doThis => {

```

Comment:
`doThis` could also be just `()`, since your are not passing any data to the Subject/EventEmitter; You can also omit the curly braces around the readContents-Method.